app/src/main/java/com/example/triptracker/model/itinerary/Itinerary.kt

```diff
@@ -22,11 +22,14 @@ data class Itinerary(
     val userMail: String,
     val location: Location,
     val flameCount: Long,
+    val saves: Long,
+    val clicks: Long,
+    val numStarts: Long, // the number of times the trip has been started
```

Comment:
I think we should add the description of these attributes in the class description comment